### PR TITLE
More Nuclei translations

### PR DESCRIPTION
**Pull request by kazet**

### CODE REVIEW

File: artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py [hunk 1]
```diff
@@ -350,6 +350,16 @@
...
+    + UPDATE_HINT,
+    "The WordPress Social Login and Register (Discord, Google, Twitter, LinkedIn) plugin for WordPress is vulnerable to authentication bypass in versions up to, and including, 7.6.4. This is due to insufficient encryption on the user being supplied during a login validated through the plugin. This makes it possible for unauthenticated attackers to log in as any existing user on the site, such as an administrator, if they know the email address associated with that user. This was partially patched in version 7.6.4 and fully patched in version 7.6.5.": "Wtyczka WordPress o nazwie WordPress Social Login and Register w wersji do 7.6.4 włącznie umożliwia atakującemu ominięcie autoryzacji i nieuprawniony dostęp administracyjny do systemu."
+    + UPDATE_HINT,
+    # There are multiple plugins with this message, therefore we won't know the plugin name here
+    "The plugin does not sanitise and escape a parameter before using it in a SQL statement via an AJAX action (available to unauthenticated users), leading to an SQL injection.": "Wykryto wtyczkę, która zawiera podatność SQL Injection, umożliwiającą atakującemu pobranie całej zawartości bazy danych."
```
Comment (bulkowy) [inline artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py:360]: Specify that it is based on AJAX Action. And if possible, include name of plugin with appending it to the string

File: artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py [hunk 2]
```diff
@@ -350,8 +350,131 @@
...
+    + UPDATE_HINT,
+    "GeoServer through 2.18.5 and 2.19.x through 2.19.2 allows server-side request forgery via the option for setting a proxy host.": "Narzędzie GeoServer w wersji do 2.18.5 i w wersjach 2.19.x do 2.19.2 zawiera podatność typu Server-Side Request Forgery umożliwiającą atakującemu komunikację z systemami w sieci lokalnej."
+    + UPDATE_HINT,
+    "The WordPress Social Login and Register (Discord, Google, Twitter, LinkedIn) plugin for WordPress is vulnerable to authentication bypass in versions up to, and including, 7.6.4. This is due to insufficient encryption on the user being supplied during a login validated through the plugin. This makes it possible for unauthenticated attackers to log in as any existing user on the site, such as an administrator, if they know the email address associated with that user. This was partially patched in version 7.6.4 and fully patched in version 7.6.5.": "Wtyczka WordPress o nazwie WordPress Social Login and Register w wersji do 7.6.5 włącznie umożliwia atakującemu ominięcie autoryzacji i nieuprawniony dostęp administracyjny do systemu."
```
Comment (bulkowy) [inline artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py:357]: `fully patched in version 7.6.5.` -> `do 7.6.4 włącznie`

Comment (bulkowy) [reply artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py:357]: `ominięcie autoryzacji` -> `ominięcie uwierzytelnienia`

File: artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py [hunk 3]
```diff
@@ -350,8 +350,131 @@
...
+    + UPDATE_HINT,
+    "ProcessMaker 3.5.4 and prior is vulnerable to local file inclusion.": "Narzędzie ProcessMaker w wersji 3.5.4 i wcześniejszych umożliwia atakującemu odczyt dowolnych plików z serwera."
+    + UPDATE_HINT,
+    "The CMNC-200 IP Camera has a built-in web server that is vulnerable to directory transversal attacks, allowing access to any file on the camera file system.": "Wykryto, że kamera CMNC-200 zawiera podatność Directory Traversal umożliwiającą odczyt dowolnych plików z dysku."
+    + UPDATE_HINT,
+    "FTP credentials were detected.": "Wykryto plik ftpsync.settings zawierający dane logowania serwera FTP.",
+    "Apache Rocketmq Unauthenticated Access were detected.": "Wykryto, że możliwy jest dostęp bez logowania do narzędzia Apache Rocketmq.",
```
Comment (bulkowy) [inline artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py:364]:
```suggestion
    "Apache Rocketmq Unauthenticated Access were detected.": "Wykryto, że dostęp  do narzędzia Apache Rocketmq nie wymaga logowania.",
```

File: artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py [hunk 4]
```diff
@@ -350,8 +350,131 @@
...
+    "Tensorflow Tensorboard was able to be accessed with no authentication requirements in place.": "Wykryto, że dostęp do narzędzia Tensorflow Tensorboard jest możliwy bez logowania.",
```
Comment (bulkowy) [inline artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py:365]:
```suggestion
    "Tensorflow Tensorboard was able to be accessed with no authentication requirements in place.": "Wykryto, że dostęp do narzędzia Tensorflow Tensorboard nie wymaga logowania.",
```

File: artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py [hunk 5]
```diff
@@ -350,8 +350,131 @@
...
+    "Nginx server is vulnerable to local file inclusion.": "Wykryto, że serwer NGINX jest skonfigurowany w sposób umożliwiający odczyt dowolnych plików z serwera.",
```
Comment (bulkowy) [inline artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py:366]:
```suggestion
    "Nginx server is vulnerable to local file inclusion.": "Wykryto, że serwer NGINX jest niepoprawnie skonfigurowany, co umożliwia odczyt dowolnych plików z serwera.",
```

File: artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py [hunk 6]
```diff
@@ -350,8 +350,131 @@
...
+    "Genie Access WIP3BVAF WISH IP 3MP IR Auto Focus Bullet Camera devices through 3.X are vulnerable to local file inclusion via the web interface, as demonstrated by reading /etc/shadow.": "Wykryto, że urządzenie Genie Access WIP3BVAF WISH IP 3MP IR Auto Focus Bullet Camera zawiera podatność Directory Traversal umożliwiającą odczyt dowolnych plików z dysku."
+    + UPDATE_HINT,
```
Comment (bulkowy) [inline artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py:368]: in this case, this device won't have more updates, according to advisory. Might include this information in translation

File: artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py (hunk 1 above)
Comment (kazet) [reply artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py]: plugin name added. info about ajax action won't be provided as it's not of much use for our intended recipients

File: artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py [hunk 7]
```diff
@@ -350,8 +350,134 @@
...
+    + WORDPRESS_UPDATE_HINT,
+    "ProcessMaker 3.5.4 and prior is vulnerable to local file inclusion.": "Narzędzie ProcessMaker w wersji 3.5.4 i wcześniejszych umożliwia atakującemu odczyt dowolnych plików z serwera."
+    + UPDATE_HINT,
+    "The CMNC-200 IP Camera has a built-in web server that is vulnerable to directory transversal attacks, allowing access to any file on the camera file system.": "Wykryto, że kamera CMNC-200 zawiera podatność Directory Traversal umożliwiającą odczyt dowolnych plików z dysku."
+    + UPDATE_HINT,
+    "FTP credentials were detected.": "Wykryto plik ftpsync.settings zawierający dane logowania serwera FTP.",
+    "Genie Access WIP3BVAF WISH IP 3MP IR Auto Focus Bullet Camera devices through 3.X are vulnerable to local file inclusion via the web interface, as demonstrated by reading /etc/shadow.": "Wykryto, że urządzenie Genie Access WIP3BVAF WISH IP 3MP IR Auto Focus Bullet Camera zawiera podatność Directory Traversal umożliwiającą odczyt dowolnych plików z dysku.",
+    "Apache Rocketmq Unauthenticated Access were detected.": "Wykryto, że dostęp  do narzędzia Apache Rocketmq nie wymaga logowania.",
```
Comment (bulkowy) [inline artemis/reporting/modules/nuclei/translations/nuclei_messages/pl_PL.py:365]:
```suggestion
    "Apache Rocketmq Unauthenticated Access were detected.": "Wykryto, że dostęp do narzędzia Apache Rocketmq nie wymaga logowania.",
```

**Review by bulkowy — DISMISSED** [review]

**Review by bulkowy — APPROVED** [review]

**Pull request merged**